app/views/unattended/provisioning_templates/snippet/kickstart_kernel_options.erb

```diff
@@ -82,6 +82,14 @@ snippet: true
 
   # VLAN (only on physical is recognized)
   if iface.virtual? && iface.tag.present? && iface.attached_to.present?
+    if iface.attached_to.match 'bond'
```

Comment:
Yeah this is the only reasonable way to find if an interface is attached to a bond. We do have `bond?` method but there is no direct database association among these. :-(